application/controllers/DashboardsController.php

```diff
@@ -0,0 +1,465 @@
+                    if ($orgHome && $orgHome->hasEntry($pane->getName()) && $home->hasEntry($pane->getName())) {
+                        Notification::error(sprintf(
+                            t('Dashboard "%s" already exists within "%s" home'),
+                            $pane->getTitle(),
+                            $home->getTitle()
+                        ));
+                        break;
+                    }
+
+                    // Perform DB updates
+                    $home->reorderWidget($pane, (int) $indexOrValues, $orgHome);
+                    if ($orgHome) {
+                        // In order to properly update the dashlets id (user + home + pane + dashlet)
+                        $pane->manageEntry($pane->getEntries());
+                    }
+
+                    Notification::success(sprintf(
+                        t('%s dashboard pane "%s" successfully'),
+                        $orgHome ? 'Moved' : 'Updated',
+                        $pane->getTitle()
+                    ));
+                    break;
+                }
+
+                foreach ($indexOrValues as $dashlet => $index) {
+                    if (! $pane->hasEntry($dashlet) && (! $orgPane || ! $orgPane->hasEntry($dashlet))) {
+                        Notification::error(sprintf(t('Dashlet "%s" not found'), $dashlet));
+                        break;
+                    }
+
+                    if ($orgPane && $orgPane->hasEntry($dashlet) && $pane->hasEntry($dashlet)) {
+                        Notification::error(sprintf(
+                            t('Dashlet "%s" already exists within "%s" dashboard pane'),
+                            $dashlet,
+                            $pane->getTitle()
+                        ));
+                        break;
+                    }
+
+                    $dashlet = $pane->hasEntry($dashlet) ? $pane->getEntry($dashlet) : $orgPane->getEntry($dashlet);
+                    $pane->reorderWidget($dashlet, (int) $index, $orgPane);
+
+                    Notification::success(sprintf(
+                        t('%s dashlet "%s" successfully'),
+                        $orgPane ? 'Moved' : 'Updated',
+                        $dashlet->getTitle()
+                    ));
+                }
+            }
+        }
+
+        $this->createTabs();
+        $this->dashboard->activate('dashboard_settings');
+        $this->sendMultipartUpdate();
+    }
+
+    /**
+     * Provides a mini wizard which guides a new user through the dashboard creation
+     * process and helps them get a first impression of Icinga Web 2.
+     */
+    public function setupDashboardAction()
+    {
+        if (isset($this->getRequest()->getPost()['btn_next'])) {
+            // Set compact view to prevent the controls from being
+            // rendered in the modal view when redirecting
+            $this->view->compact = true;
+
+            $this->getResponse()->setHeader('X-Icinga-Title', t('Configure Dashlets'), true);
```

Comment:
You can use `setTitle()` here now. It will work just fine once you've rebased with master.